features/step_definitions/events_newsletter_steps.rb

```diff
@@ -0,0 +1,17 @@
+When(/^"([^"]*)" opens the email$/) do | user|
+  user = User.where(superadmin: false)
+end
+
+Then(/^I should see email delivered from "([^"]*)"$/) do |sender|
+  expect(email).to have_sender(sender)
```

Comment:
so email here will have to de defined, or be an instance variable or passed in as an argument

Reply:
okay will look into that